app/brave_generated_resources.grd

```diff
@@ -469,8 +469,119 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_BRAVE_SYNC_TITLE" desc="The title for Brave Sync in settings">
         Sync
       </message>
-      <message name="IDS_SETTINGS_BRAVE_SYNC_LINK_LABEL" desc="Brave Sync link label">
-        Access Sync via
+      <message name="IDS_SETTINGS_BRAVE_SYNC_SETUP_ACTION_LABEL" desc="Brave Sync setup action label">
+        Start using sync
+      </message>
+      <message name="IDS_SETTINGS_BRAVE_SYNC_SETUP_TITLE" desc="Title for the Brave Sync settings setup section">
+        Sync Setup
+      </message>
+      <message name="IDS_SETTINGS_BRAVE_SYNC_SETUP_SUBTITLE" desc="Subtitle for the Brave Sync settings setup section">
+        To start, you will need Brave installed on all the devices you plan to sync. Then, securely link them together with a sync code.
+      </message>
+      <message name="IDS_SETTINGS_BRAVE_SYNC_MANAGE_ACTION_LABEL" desc="Brave Sync manage action label">
+        Manage your sync'd devices
```

Comment:
i think `synced` instead of `sync'd` looks more correct?

Reply:
FYI @bradleyrichter `sync'd` -> `synced`